File: app/Flecha Digital/Model/Actividad.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flecha_Digital.Model
{
    public class Actividad
    {
        public int no_operacion { get; set; }
        public int origen_viaje { get; set; }
        public int destino_viaje { get; set; }
        public DateTime fecha_salida { get; set; }
        public IEnumerable Boleto { get; set;}
    }
}

File: server-legacy/services/reservas.api/models/classes/actividad.js
class Actividad {
    constructor(actividad) {
        this.id_usuario = actividad.id_usuario;
        this.no_operacion = actividad.no_operacion;
        this.origen_viaje = actividad.origen_viaje;
        this.destino_viaje = actividad.destino_viaje;
        this.fecha_salida = actividad.fecha_salida;
        this.fecha_llegada = actividad.fecha_llegada;
        this.cantidad_boletos = actividad.cantidad_boletos;
    };
}

module.exports = Actividad;